tests/zkevm/test_worst_compute.py

```diff
@@ -104,3 +104,73 @@ def test_worst_keccak(
         post={},
         blocks=[Block(txs=[tx])],
     )
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        36_000_000,
+    ],
+)
+def test_worst_modexp(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_limit: int,
+):
+    """Test running a block with as many MODEXP calls as possible."""
+    env = Environment(gas_limit=gas_limit)
+
+    base_mod_length = 32
+    exp_length = 32
+
+    base = 2 ** (8 * base_mod_length) - 1
+    mod = 2 ** (8 * base_mod_length) - 2  # Prevents base == mod
+    exp = 2 ** (8 * exp_length) - 1
+
+    # MODEXP calldata
+    calldata = (
+        Op.MSTORE(0 * 32, 32)
+        + Op.MSTORE(1 * 32, 32)
+        + Op.MSTORE(2 * 32, 32)
```

Comment:
```suggestion
        Op.MSTORE(0 * 32, base_mod_length)
        + Op.MSTORE(1 * 32, exp_length)
        + Op.MSTORE(2 * 32, base_mod_length)
```
We could reuse the variables above.